flow360/component/simulation/translator/solver_translator.py

```diff
@@ -616,6 +619,23 @@ def boundary_spec_translator(model: SurfaceModelTypes, op_acousitc_to_static_pre
     return boundary
 
 
+def set_aft_ncrit(ncrit_input, turb_intensity_percent_input):
```

Comment:
Can you instead put this as a after model validator for transitionModelSovler? The translator would be best not to process param too much as it is designed to be disposable. It should only contain logics to "translate",